jsettlers.graphics/src/jsettlers/graphics/localization/Labels.java

```diff
@@ -131,4 +131,9 @@ public static String getName(EActionType action) {
 	public static String getProgress(EProgressState loading) {
 		return getString("progress_" + loading);
 	}
+
+	public static String getName(EMaterialType material) {
+		// TODO Auto-generated method stub
```

Comment:
Anytime you want. This function is already included in the map editor. You need to move it from the map editor to graphics (and move the labels)